client/renderSDL/RenderHandler.cpp

```diff
@@ -55,6 +55,58 @@ std::shared_ptr<CDefFile> RenderHandler::getAnimationFile(const AnimationPath &
 	return result;
 }
 
+std::optional<ResourcePath> RenderHandler::getPath(ResourcePath path)
```

Comment:
Perhaps we should clean up this logic? So:
- path MUST be specified without `Data` or `Sprites` prefix
- for unscaled images `Sprites` is preferred, `Data` is only used as fallback location
- for animations, always load them from `Sprites`